packages/itwinui-react/src/core/Avatar/Avatar.tsx

```diff
@@ -99,11 +99,8 @@ export const Avatar = React.forwardRef((props, ref) => {
   return (
     <Box
       as='span'
-      className={cx(
-        'iui-avatar',
-        { [`iui-${size}`]: size !== 'medium' },
-        className,
-      )}
+      className={cx('iui-avatar', className)}
+      data-iui-size={size}
```

Comment:
Based on the CSS, `data-iui-size` doesn't seem to be needed when `size === "medium"`. So we should likely add the data attribute only when `size !== "medium"`.

Same for `AvatarGroup`.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1935/commits/9bf284d63c2559846c30a87f20c757e1d6647d67) Also updated unit tests.